### PR TITLE
Added new items since v1.3

### PR DESCRIPTION
**Pull request by cjakeman**
Also changed section title so that it now makes sense to add items to the list as they are added to the manual.
See 
http://www.elvastower.com/forums/index.php?/topic/34924-whats-new-in-open-rails/page__pid__269154#entry269154

### CODE REVIEW

**Review by YoRyan — APPROVED** [review]
Syntax looks good, as does your list of additions. Are we tentatively ready for 1.4?

File: Source/Documentation/Manual/news.rst [hunk 1]
```diff
@@ -1,61 +1,49 @@
...
-- Content-creator timetable options for random delays on various actions
-- Expose to users internal option to force all objects to cast shadows
-- Content-creator option for timetables to override default stopping time at stations
-- Better checks for when trains stop at stations
-- Simulation of wind resistance forces
-- Option to automatically discard abnormal values in locomotives
+- Vacuum brakes, EP braking and air braking
+- Train control system (TCS)
+- Visual effects for steam ejectors
+- Water troughs
```
Comment (YoRyan) [inline Source/Documentation/Manual/news.rst:46]:
```suggestion
- Water troughs
- Screenshot button now plays a camera shutter sound
```

**Comment by cjakeman** [comment]
Hi Ryan,
 
> Syntax looks good, as does your list of additions. Are we tentatively ready for 1.4?

Thanks.

I was trying to get ready for v1.4 ahead of time, but this list should really be updated with each improvement to stay in step as changes are made to the manual.

It's been a long time since v1.3, so I reckon we ought to release as soon as the Monogame merge is stable. But also, there are a lot of changes that have made it into OR_NewYear_MG and are not present in the official version. I think it might take a while to bring those in and I would prefer to release v1.4 first.

What do you think?

Best wishes

**Comment by YoRyan** [comment]
> It's been a long time since v1.3, so I reckon we ought to release as soon as the Monogame merge is stable. But also, there are a lot of changes that have made it into OR_NewYear_MG and are not present in the official version. I think it might take a while to bring those in and I would prefer to release v1.4 first.

I agree. There are a lot of features exclusive to the NewYear fork and it could well take a few more years to mainline all of them. Realistically, I believe the two forks will coexist for the foreseeable future.

@Csantucci also prioritizes the rapid delivery of new features. That means accepting whatever patches get shared on Elvas Tower--which keeps players happy, but bypasses the usual QA and peer review steps we do around here. If we try to mainline his changes, that could entail making yet more changes to bring them in line with OR standards. Like #369, which is waiting on a redesign to use JSON.

Some popular NewYear features have also been outright rejected by @twpol, like the [ability to scroll](https://trello.com/c/6l1RpVOr/124-scroll-extended-f5-information-for-long-trains-and-ability-to-highlight-a-line) the F5 extended HUD.

**Review by YoRyan — APPROVED** [review]

**Pull request merged**